docs/upgrade-guide.md

```diff
@@ -2,6 +2,28 @@
 
 ## Upgrading to v9.1
 
+### Uniform buffer objects
```

Comment:
Do we really want to lead with this? This is kind of advanced topic for users that write their own shaders?

Reply:
Moved to bottom